config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleMetadataBuilder.java

```diff
@@ -61,6 +61,7 @@ public BundleMetadata build(final AnnotatedBundle annotatedBundle, final Bundle
             builder.environmentEntities(getEnvironmentDependenciesMetadata(dependentEntities));
             builder.dependencies(annotatedBundle.getDependentBundles());
             builder.tags(annotatedEntity.getTags());
+            builder.portalTemplate(Boolean.valueOf(encass.getProperties().get("portalTemplate").toString()));
```

Comment:
You can use `PORTAL_TEMPLATE` constant here instead of String literal.

Reply:
done